aztec/src/main/java/org/wordpress/aztec/Html.java

```diff
@@ -520,6 +522,12 @@ private static void end(SpannableStringBuilder text, AztecTextFormat textFormat)
 
         int where = text.getSpanStart(span);
         text.setSpan(span, where, text.length(), Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
+        //
+        // Process HTML style attribute
+        if (span instanceof IAztecAttributedSpan) {
```

Comment:
The type check is redundant, since `IAztecInlineSpan` extends `IAztecAttributedSpan`.